libpldmresponder/fru.cpp

```diff
@@ -2,6 +2,10 @@
 
 #include "common/utils.hpp"
 
+#ifdef OEM_IBM
+#include "oem/ibm/libpldmresponder/utils.hpp"
```

Comment:
what reference you are providing from oem layer to here? please check if that reference can be moved to non oem layer.

Reply:
+1

Reply:
addressed this